python/ray/train/tests/test_torch_transformers_train.py

```diff
@@ -34,7 +34,7 @@ def ray_start_8_cpus():
 
 
 # We are only testing Causal Language Modeling here
-MODEL_NAME = "hf-internal-testing/tiny-random-gpt2"
+MODEL_NAME = "hf-internal-testing/tiny-random-BloomForCausalLM"
```

Comment:
I do not really understand this change..

Reply:
this is @matthewdeng 's change